libspawner/src/win32/runner.cpp

```diff
@@ -252,8 +252,7 @@ void runner::create_process() {
             si.hStdError = streams[std_stream_error]->get_pipe()->get_output_handle();
     }
 
-    //FIXME: C++11 forbids implicit conversion of a string constant to 'char*'
-    si.lpDesktop = "";
```

Comment:
It fixes aforementioned issue and it's the approach that used in `init_process_with_logon()`.
If even running from other user inherits desktop from parent process, why then we need to create new desktop at simple process creation?